pages/common/git-sed.md

```diff
@@ -0,0 +1,21 @@
+# git sync
```

Comment:
```suggestion
# git sed
```

Reply:
I think we can apply this small fix then merge this.

Reply:
That completely slipped me, I'm sorry. Thanks for pointing it out.